### PR TITLE
feat: add reveal guide

### PR DESCRIPTION
**Pull request by lcschy**
## Description

- Add guide for revealing tokenized data with Elements

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

- [x] Not Applicable

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: docs/guides/share/reveal-tokenized-data.mdx [hunk 1]
```diff
@@ -0,0 +1,882 @@
...
+</Alert>
+
+<Alert>
+  Save the API Key from the created Private Application as it will be used later in this guide.
+</Alert>
+
+### Authorizing In the Backend
+
+Now we can use the created Private Application API Key, and the `nonce` from our created session to authorize it.
+We'll also use a [Condition](/docs/concepts/access-controls#conditions) to make sure our session is only authorized to reveal our desired token.
```
Comment (armsteadj1) [inline docs/guides/share/reveal-tokenized-data.mdx:375]:
```suggestion
Using the Private Application API Key, the `nonce` from our session, and [Condition](/docs/concepts/access-controls#conditions) we ensure we are only granting authorization for the desired token.
```

File: docs/guides/share/reveal-tokenized-data.mdx [hunk 2]
```diff
@@ -0,0 +1,882 @@
...
+
+This step <b>SHOULD</b> be done on the backend as to not expose the Private Application's API Key publicly.
```
Comment (armsteadj1) [inline docs/guides/share/reveal-tokenized-data.mdx:377]:
```suggestion
The *only way* to ensure your private API keys are not publically accessible is for this step to execute in your backend service.
```

File: docs/guides/share/reveal-tokenized-data.mdx [hunk 3]
```diff
@@ -0,0 +1,882 @@
...
+  response.json({
+    result: "success",
+  });
+});
+app.listen(port, () => {
+  console.log(`Example app listening on port ${port}`);
+});
+```
+
+Now we can start the server with the following command (from the same directory as `backend.js`).
```
Comment (armsteadj1) [inline docs/guides/share/reveal-tokenized-data.mdx:421]:
```suggestion
Start the server with the following command (from the same directory as `backend.js`).
```

File: docs/guides/share/reveal-tokenized-data.mdx [hunk 4]
```diff
@@ -0,0 +1,882 @@
...
+}
+```
+  </TabItem>
+</Tabs>
+
+🎉 The code above is the last bit that we need to reveal a token! 
+
+## Conclusion
+
+You can now reveal any data to a customer without your iOS app accessing the underlying value, drastically reducing compliance and regulatory scope.
```
Comment (armsteadj1) [inline docs/guides/share/reveal-tokenized-data.mdx:875]:
```suggestion
You can now reveal any data to a customer without your applications accessing the underlying value, reducing compliance and regulatory scope.
```

**Review by armsteadj1 — CHANGES_REQUESTED** [review]

File: docs/guides/share/reveal-tokenized-data.mdx (hunk 4 above)
Comment (lcschy) [reply docs/guides/share/reveal-tokenized-data.mdx:875]: oopsie

File: docs/guides/share/reveal-tokenized-data.mdx [hunk 5]
```diff
@@ -0,0 +1,884 @@
...
+            }
+      }
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+    }
+
+    //highlight-start
```
Comment (brigonzalez) [inline docs/guides/share/reveal-tokenized-data.mdx:779]: we want to remove this highlight right? since we've already highlighted this in the step before?

**Review by brigonzalez — APPROVED** [review]

**Review by armsteadj1 — APPROVED** [review]

**Pull request merged**